lib/Doctrine/ORM/Configuration.php

```diff
@@ -54,6 +54,9 @@
  */
 class Configuration extends \Doctrine\DBAL\Configuration
 {
+    /** @var mixed[] */
+    protected $_attributes = []; /** @codingStandardsIgnoreLine */
```

Comment:
Please use an ignore rule in `phpcs.xml.dist` for this.

```suggestion
    protected $_attributes = [];
```

Reply:
@greg0ire - thanks, changed. :)